clickhouse_sqlalchemy/alembic/comparators.py

```diff
@@ -35,10 +35,12 @@ def _extract_to_table_name(create_table_query):
 
 
 def _reflect_table(inspector, table):
-    if alembic_version >= (1, 11, 0):
-        return _alembic_reflect_table(inspector, table)
+    if alembic_version >= (1, 15, 0):
+        return inspector.reflect_table(table, None)
+    elif alembic_version >= (1, 11, 0):
+        return sqla_compat._reflect_table(inspector, table)
     else:
-        return _alembic_reflect_table(inspector, table, None)
+        return sqla_compat._reflect_table(inspector, table, None)
```

Comment:
```suggestion
    if alembic_version >= (1, 11, 0):
        return sqla_compat._reflect_table(inspector, table)
    return sqla_compat._reflect_table(inspector, table, None)
```
Just `if` is enough in lieu of an `elif` and then `else` clause because of immediate `return` statements.